src/parser/ad_parser.js

```diff
@@ -237,47 +236,70 @@ function parseWrapper(wrapperElement) {
  */
 function parseExtensions(collection, extensions) {
```

Comment:
Also the typings can be improved from `Array` to `Node[]` for `extensions` param